### ISSUE TITLE
Include Github commit hash in --version output

### ISSUE THREAD

**Issue by zabil** [getgauge/taiko#545]
When installing Taiko using 
```
npm install https://github.com/getgauge/taiko 
```

It is important to know which commit from master is installed to easily track issues and bugs. To do this modify `taiko --version` to include the commit hash. for example

```
$ npm install https://github.com/getgauge/taiko 
$ taiko --version
Version: 0.7.0 (Chromium: 74.0.3723.0) - 06135731a9c1175bed934d089ac477b698c18e94
```

or when a specific version is installed

```
$ npm install https://github.com/getgauge/taiko#9918f72eaabef8af1312c0d791281067
$ taiko --version
Version: 0.7.0 (Chromium: 74.0.3723.0) - 9918f72eaabef8af1312c0d791281067
```

**Comment by NivedhaSenthil** [comment]
Fix available in e2c6a64. Currently it wont give hash for installation done using local repo `npm install -g .`

**Comment by NivedhaSenthil** [comment]
Seems like this has introduced delay in `taiko` runner and repl as printVersion method is called irrespective of `-v` every time.

**Comment by NivedhaSenthil** [comment]
Fix available in d9bc471. Still it wont give hash for installation done from local repo `npm install -g .`

**Comment by Debashis9012** [comment]
This enhancement has been verified and found fixed with the version ```d9bc471```
